### DISCUSSION TITLE
get raw table data

### DISCUSSION THREAD

**Discussion by MarcDAFrame** [remarkjs/remark#546] [Q&A] [answered]
```
<Markdown 
    renderers={{
        table: Table,
    }}
/>
const Table = (props) => {
    console.log(props)
    return null
}
```

and the props on table are 
```
{ children: [{
    $$typeof: Symbol(react.element),
    key: "tableHead-3-1-0"
    props: {
        children: [{
            $$typeof: Symbol(react.element),
            key: "tableRow-3-1-0"
            props: {
                children: [
                    {$$typeof: Symbol(react.element), key: "table-... }
                    ...
```

is there any way to get the props as either the string representation of the table or as an array in this format
```
| Date | Value |
| -- | -- |
| Oct 9 2020 12:30 | 1200 |
```
or
```
[
   [ "Date", "Value"],
   [ "Oct 9 2020 12:30", "1200"],
]
```

Thanks in advance!

**Comment by wooorm** [comment, marked as answer]
a) this sounds like an XY problem: what do you want to achieve in the end?
b) you also have access to `node`, which is a way into the [mdast](https://github.com/syntax-tree/mdast) syntax tree. You can [traverse the tree](https://unifiedjs.com/learn/recipe/tree-traversal/) to get an array form out of it

**Reply by MarcDAFrame** [reply]
I'm hoping to use [handsontable](https://www.npmjs.com/package/@handsontable/react) or someother data-grid for rendering and editing of the markdown table.

Thanks for these resources I'll look into them.
